src/components/SettingItem.tsx

```diff
@@ -0,0 +1,28 @@
+import { styled } from "linaria/react"
+
+const Title = styled.div`
+  font-size: 1.8rem;
+  font-weight: 500;
+`
+
+const Subtitle = styled.div`
+  color: var(--grey-color);
+  font-size: 1.4rem;
+`
+
+interface Props {
+  title: string
+  subtitle?: string
+  href?: string
```

Comment:
Should href be optional?

Reply:
Yeah just checked this is fine, for example, there's a version number in the settings which shouldn't be clickable

Reply:
Maybe this is bad UX but it's minor, something to look at in the future

Reply:
Yeah thought about adding a different tag for settings that aren't clickable but when href is null it doesn't get added to the `<a>`  tag and so isn't clickable anyway

Reply:
That's a good point actually if there's no href we shouldn't render an anchor tag anyway, I think that would confuse screen readers could do to simply a better solution:

```jsx
const Container = styled.a

return (
  <Container as={href ? "a" : "div"}>
    ...
  </Container>
)
```